mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/BaseHomeContract.java

```diff
@@ -17,8 +17,6 @@
 public interface BaseHomeContract {
 
     interface BaseHomeView extends BaseView<BaseHomePresenter> {
-
-        void showClientDetails(Client client);
     }
 
     interface BaseHomePresenter extends BasePresenter {
```

Comment:
@devansh-299 by chance if we removed both of the interfaces (`BaseHomeView` , `BaseHomePresenter`).
then where we add `void fetchClientDetails();` method??